examples/flutter_firebase_login/lib/sign_up/cubit/sign_up_cubit.dart

```diff
@@ -17,15 +18,28 @@ class SignUpCubit extends Cubit<SignUpState> {
     final email = Email.dirty(value);
     emit(state.copyWith(
       email: email,
-      status: Formz.validate([email, state.password]),
+      status: Formz.validate([email, state.password, state.confirmedPassword]),
     ));
   }
 
   void passwordChanged(String value) {
     final password = Password.dirty(value);
+    final confirmedPassword = ConfirmedPassword.dirty(
+        original: password, value: state.confirmedPassword.value);
```

Comment:
```suggestion
        original: password, value: state.confirmedPassword.value,);
```

nit but can we add a trailing comma and format for improved readability?